resource_generator/templates/check-page-per-org.html

```diff
@@ -38,37 +38,48 @@
 <span class="govuk-caption-xl">Brownfield land</span>
 <h1 class="govuk-heading-xl">Check your data</h1>
 <p class="govuk-body">Check that the brownfield land data we have collected from {{ organisation|map_org_code_to_name }} is accurate and up to
-    date.</p>
+    date by following the steps on this page.</p>
 
-<h2 class="govuk-heading-m">Latest version</h2>
-<p class="govuk-body">This is the most recent version of your brownfield land register that we have collected.</p>
+<h2 class="govuk-heading-m">1. Make sure we have your most recent register</h2>
+<p class="govuk-body">This is the most recent version of your brownfield land register that we have collected:</p>
 <div class="highlight-box--cta">
-    <a href="/resource/{{ latest_resource['resource'] }}">Version last collected on
+    View the data: <a href="/resource/{{ latest_resource['resource'] }}">version last collected on
         {{ latest_resource['last_collected'] }}</a>
 </div>
+<p class="govuk-body">If you have a more up-to-date register than the above version, it may be linked to at the bottom of this page. If it is not, you can see how the data in your most up-to-date register looks by running it through the View the data page.</p>
```

Comment:
<p class="govuk-body">If you have a more up-to-date register than the above version, please let us know using <a href="https://forms.gle/YoxWZf2tjaRxKHCAA">via this form</a> or emailing us at DigitalLand@communities.gov.uk.</p>